terrapower/physics/neutronics/dragon/dragonExecutor.py

```diff
@@ -204,3 +190,6 @@ def executeDragon():
             outputPaths,
             executeDragon,
         )
+
+
+from .dragonFactory import dragonFactory
```

Comment:
Is there a reason this is imported at the bottom? Agree with Mitch on his factory comments.

Reply:
circular imports I'm sure. should comment though